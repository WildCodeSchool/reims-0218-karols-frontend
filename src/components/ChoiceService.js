import React from "react"
import {
  Card,
  CardText,
  CardBody,
  CardTitle,
  Button,
  CardFooter
} from "reactstrap"

const ChoiceService = ({ name, description, image }) => {
  return (
    <div>
      <Card>
        <CardBody>
          <CardTitle
            style={{
              "font-size": "2em"
            }}
          >
            {name}
          </CardTitle>
        </CardBody>
        <a href="">
          <img
            className="img-fluid"
            width="250"
            height="250"
            src={image}
            alt="Name of service"
          />
        </a>
        <CardBody>
          <CardText
            style={{
              "font-size": "1.5em"
            }}
          >
            {" "}
            {description}{" "}
          </CardText>
        </CardBody>
        <CardFooter>
          <Button> Choisir ce service </Button>
        </CardFooter>
      </Card>
    </div>
  )
}

export default ChoiceService
