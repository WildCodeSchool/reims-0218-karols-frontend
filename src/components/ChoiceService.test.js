import React from "react"
import ReactDOM from "react-dom"
import ChoiceService from "./ChoiceService"
import renderer from "react-test-renderer"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<ChoiceService />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders correctly", () => {
  const tree = renderer.create(<ChoiceService />).toJSON()
  expect(tree).toMatchSnapshot()
})
