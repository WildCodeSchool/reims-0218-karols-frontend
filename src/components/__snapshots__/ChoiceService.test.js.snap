// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="card text-white"
  >
    <img
      className="card-img"
      src={undefined}
      width="100%"
    />
    <div
      className="card-img-overlay"
    >
      <h5
        className="card-title"
        style={
          Object {
            "fontSize": "2em",
          }
        }
      />
      <p
        className="card-text"
        style={
          Object {
            "fontSize": "1.5em",
          }
        }
      />
    </div>
    <div
      className="card-footer"
    >
      <button
        className="btn btn-secondary"
        onClick={[Function]}
        type={undefined}
      >
         Choisir ce service 
      </button>
    </div>
  </div>
</div>
`;
